src/Generator/AttributeGenerator.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Code\Generator;
+
+use Laminas\Code\Generator\AttributeGenerator\AttributeAssembler;
+use Laminas\Code\Generator\AttributeGenerator\AttributeAssemblerBag;
+use Laminas\Code\Generator\AttributeGenerator\AttributeAssemblerFactory;
+use Laminas\Code\Generator\AttributeGenerator\AttributeBuilder;
+use ReflectionClass;
+
+class AttributeGenerator extends AbstractGenerator
```

Comment:
We should probably make this `final`, since it's a new symbol, and inheritance abuse is strong :-)

Reply:
Yes, I don't know why I extended it, maybe in the beginning when I didn't know the code, I'll stay on with interface `GeneratorInterface` :)